rust_snuba/src/strategies/python.rs

```diff
@@ -222,8 +222,7 @@ impl ProcessingStrategy<KafkaPayload> for PythonTransformStep {
                 partition,
                 timestamp,
             }) => {
-                // TODO: Handle None payload
-                let payload_bytes = (payload.payload().unwrap()).clone();
+                let payload_bytes = payload.payload().cloned().unwrap_or_default();
```

Comment:
is there a way to early-return rather than to pass an empty `Vec` through python which will then either early-return or error anyway?

Reply:
We could early return here, but I'm not sure it's worth adding this optimization -- I think it's easier to let the Python code deal with it like any other invalid message and let the message be DLQ'ed and it's offsets committed as normal. It's also technically valid so if I guess someone wanted to write a Python processor with some headers/key and no payload then this way lets them do it (even though that doesn't necessarily make much sense in this particular scenario).